dav-controller/api/templates/verified_credentials.html

```diff
@@ -0,0 +1,500 @@
+
+          // Turn off the spinner
+          qrcode.classList.remove("pending");
+          // Verified picture
+          verifiedpic.style.display = "none";
+          break;
+        case "verified":
+          // Header elements
+          intro.style.display = "none";
+          success.style.display = "grid";
+          pending.style.display = "none";
+          failed.style.display = "none";
+          expired.style.display = "none";
+
+          // Button elements
+          refreshButton.style.display = "none";
+          deeplinkButton.style.display = "flex";
+          scannedMask.style.display = "flex";
+
+          // Turn off the spinner
+          qrcode.classList.remove("pending");
+          // Verified picture
+          verifiedpic.style.display = "flex";
+          break;
+        case "failed":
+          // Header elements
+          intro.style.display = "none";
+          success.style.display = "none";
+          pending.style.display = "none";
+          failed.style.display = "grid";
+          expired.style.display = "none";
+
+          // Button elements
+          refreshButton.style.display = "flex";
+          deeplinkButton.style.display = "flex";
+          scannedMask.style.display = "none";
+
+          // Turn off the spinner
+          qrcode.classList.remove("pending");
+          // Verified picture
+          verifiedpic.style.display = "none";
+          break;
+        case "pending":
+          // Header elements
+          intro.style.display = "none";
+          success.style.display = "none";
+          pending.style.display = "grid";
+          failed.style.display = "none";
+          expired.style.display = "none";
+
+          // Button elements
+          refreshButton.style.display = "none";
+          deeplinkButton.style.display = "flex";
+          scannedMask.style.display = "flex";
+
+          // Turn on the spinner
+          qrcode.classList.add("pending");
+          // Verified picture
+          verifiedpic.style.display = "none";
+          break;
+        case "expired":
+          // Header elements
+          intro.style.display = "none";
+          success.style.display = "none";
+          pending.style.display = "none";
+          failed.style.display = "none";
+          expired.style.display = "grid";
+
+          // Button elements
+          refreshButton.style.display = "flex";
+          deeplinkButton.style.display = "none";
+          scannedMask.style.display = "none";
+
+          // Turn off the spinner
+          qrcode.classList.remove("pending");
+          // Verified picture
+          verifiedpic.style.display = "none";
+          break;
+        }
+    };
+
+    document.getElementById("refresh-button").addEventListener("click", () => {
+      location.reload(true);
+    });
+    document.getElementById("deep-link-btn").addEventListener("click", () => {
+      window.open("{{deep_link_url}}", '_blank').focus();
+    });
```

Comment:
I realized today we may need to flex on the deep link type based on OOB/connectionless proof-request - I will add an issue to deal with this later.